### PR TITLE
Site Settings: update JP Disconnect UX Survey with follow-up actions.

### PR DESCRIPTION
**Pull request by AnnaMag**
**Reference PR** used to bootstrap ideas. The design diverged substantially.

Complete the JP Disconnect UX Survey with the follow-up actions.

Bare-bones design demonstrating actions performed on selecting Survey Q1 for different plans. Pending design mockups.

**Depends on**:
https://github.com/Automattic/wp-calypso/pull/17962 

**To Test**:
 -  Checkout this branch, or use calypso.live
    https://calypso.live/?branch=update/site-settings-disconnect-site-survey-follow-up

For Jetpack and non-Atomic sites:
  -  verify that the options are display as shown
 
All other sites: have no access = no site visibility. 
Direct access redirects to `/settings/general:site`, where `:site` is a Jetpack site.

in progress

**Visuals**:
Free plan:

screen shot 2017-09-10 at 14 49 19

Paid plans:

screen shot 2017-09-10 at 23 38 50

All plans:

screen shot 2017-09-10 at 20 49 59

For Premium/Pro users:

screen shot 2017-09-10 at 14 19 52

### CODE REVIEW

File: client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+							options={ options }
+							selectedText={ optionValue.value }
+							initialSelected={ 'Backups' }
+						/>
+					);
+				}
+				if ( isFreeJetpackPlan( site.plan ) ) {
+					return (
+						<h1>
+							{ translate( 'Feel free to {{a}}get in touch{{/a}} with our Happiness Engineers.', {
```
Comment (a8ci18n) [inline client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'No problem! Feel free to {{a}}get in touch{{/a}} with our Happiness Engineers.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+	getFollowUpQuestion = optionValue => {
+		const { site, translate } = this.props;
+
+		let textShareWhat = '';
+
+		switch ( optionValue ) {
+			case 'tooHard':
+				if ( isFreeJetpackPlan( site.plan ) ) {
+					textShareWhat = translate( 'Can we help ? ' );
```
Comment (a8ci18n) [inline client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'How can we help?' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx (hunk 1 above)
Comment (AnnaMag) [reply client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx]: split translatable string is fixed

File: client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx (hunk 2 above)
Comment (AnnaMag) [reply client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx]: Questions/ notes :
-  should we remove `Skip Survey` in the last screen?There is a way out of the Flow already,
or keep it for consistency?

- `What features were you looking for?` is meant as an autocomplete search, so the component will resemble the Search action for `Themes`. Here the simple search is a placeholder.

**Comment by AnnaMag** [comment]
Questions/ notes :
-  should we remove `Skip Survey` in the last screen?There is a way out of the Flow already,
or keep it for consistency?

- `What features were you looking for?` is meant as an autocomplete search, so the component will resemble the Search action for `Themes`. Here the simple search is a placeholder.

File: client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx (hunk 2 above)
Comment (AnnaMag) [reply client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx]: but that might be a better choice of a questions, actually :)

**Comment by keoshi** [comment]
Here's a mock for a Q2 format:

02-survey-step-q2

I think we should keep these as consistent among themselves as possible. This means always having this multiple line list format, and presenting the user with an input only in edge cases where we either have too many options (auto-complete), or it's a purely open ended answer.

More context here: p7rd6c-Sq-p2

**Comment by johnHackworth** [comment]
Since this is a reference PR, I'm closing it to avoid any confusion

**Pull request closed without merging**